cluster/gce/gci/configure.sh

```diff
@@ -403,6 +415,10 @@ function install-kube-binary-config {
   # Install crictl on each node.
   install-crictl
 
+  if [[ "${KUBERNETES_MASTER:-}" == "false" ]]; then
```

Comment:
if you are going to inject EXEC_AUTH_PLUGIN_* in kube-env, it needs to be below line 422

Reply:
Can you elaborate why?

Reply:
That's where we source the kube-env. Before that, the parameters aren't loaded in the bash env.

Reply:
This block is in a function `install-kube-binary-config` which gets called after line 422.
Rest of this function is also using env vars from kube-env.

Reply:
ack, thanks.